rhacs-operator/tasks/main.yml

```diff
@@ -0,0 +1,106 @@
+---
+- name: Setting Up Results Var
+  ansible.builtin.set_fact:
+    test_result: "pass"
+# 1. Testing RHACS operator
+- name: Testing Red Hat Advanced Cluster Security
+  block:
+    - name: Create stackrox namespace
+      kubernetes.core.k8s:
+        name: stackrox
+        api_version: v1
+        kind: Namespace
+        state: present
+    - name: Create stackrox central services
+      kubernetes.core.k8s:
+        name: stackrox-central-services
+        namespace: stackrox
+        api_version: platform.stackrox.io/v1alpha1
+        kind: Central
+        definition:
+          metadata:
+            labels:
+              opdev: opcap
+          spec:
+            central:
+              exposure:
+                route:
+                  enabled: true
+        wait: true
+        wait_condition:
+          status: "True"
+          type: "Deployed"
+        wait_timeout: 90
+        state: present
+    - name: Get Central Secret Password
+      ansible.builtin.shell: |
+        /usr/local/bin/oc -n stackrox get secret central-htpasswd -o go-template={% raw %}'{{index .data "password" | base64decode}}'{% endraw %}
+      register: password
+    - name: Get Central Endpoint
+      kubernetes.core.k8s_info:
+        namespace: stackrox
+        name: central
+        api_version: v1
+        kind: Route
+      register: route
+    - name: Wait for central service to be fully available
+      ansible.builtin.wait_for:
+        timeout: 60
+    - name: Create and apply init-bundle
+      ansible.builtin.shell: |
+        roxctl central init-bundles generate -e https://{{ route.resources[0].spec.host }}:443 --insecure-skip-tls-verify -p {{ password.stdout }} main --output-secrets - | oc apply -f -
```

Comment:
Don't use the | to oc. Capture it in a registered variable, and do a d separate apply with kubernetes.core.k8s module.

Reply:
That's doable but as far as I can tell we can't escape from creating a file. I did that. Let me know if it works.

Reply:
We can get by without creating a file. I'll try it to double check.

Reply:
Let me know if you still want that without creating the file. I did some research with no success but I can dig deeper to see if I find something.

Reply:
Never mind just saw your suggestion and committed it. I didn't know about the yaml all function.